README.md

```diff
@@ -84,18 +84,20 @@ bin/rails g
 
 ## Development
 
-We use the 2 embeded Rails apps [Rails 5 Example](./recreate_rails5_example) and [Rails 6 Example](./recreate_rails6_example) to test the usage of the generators.
+Install the versions of Node and Ruby specified in `.node-version` and `.ruby-version` on your machine. https://asdf-vm.com/ is a great tool for managing language versions. Then run `npm install -g yarn`.
 
-`recreate_rails6_example` is simply a fresh generated Rails 6 app. To recreate it use:
+We use the embeded Rails apps [Rails 7 Example](./recreate_rails7_example) to test the usage of the generators.
+
+`recreate_rails7_example` is simply a freshly generated Rails 7 app. To recreate it use:
 
 ```shell
-bin/recreate_rails6_example
+bin/recreate_rails7_example
 ```
 
-Because Webpacker has so many moving parts, we want test bed where we can see the full [rolemodel:webpacker](./lib/generators/rolemodel/webpacker). To recreate it (after a version change, etc) use:
+Because Webpack has so many moving parts, we want a test bed where we can see the full [rolemodel:webpack](./lib/generators/rolemodel/webpack). To recreate it (after a version change, etc) use:
 
 ```shell
-bin/recreate_webpacker_example
+bin/copy_webpack_templates
```

Comment:
This whole section is irrelevant since we're not using Webpacker anymore. Remove it and `bin/copy_webpack_templates` script.